pkg/nodeconfig/nodeconfig.go

```diff
@@ -165,8 +165,8 @@ func NewNodeConfig(clientset *kubernetes.Clientset, clusterServiceCIDR, vxlanPor
 // For example: https://api-int.abc.devcluster.openshift.com:6443 gets translated to
 // api-int.abc.devcluster.openshift.com
 // TODO: Think if this needs to be removed as this is too restrictive. Imagine apiserver behind
-// 		a loadbalancer.
-// 		Jira story: https://issues.redhat.com/browse/WINC-398
```

Comment:
Can this be made to line up with the `Think`?

Reply:
We should be consistent about lining up. We are doing things differently for `Note:`.

Reply:
no bringing it inline with think is not making linter happy.

Here is an example from: https://go.dev/doc/comment

Even with these heuristics, other existing comments will need manual adjustment to correct their rendering. The most common mistake is indenting a wrapped unindented line of text. For example:

// TODO Revisit this design. It may make sense to walk those nodes
//      only once.

// According to the document:
// "The alignment factor (in bytes) that is used to align the raw data of sections in
//  the image file. The value should be a power of 2 between 512 and 64 K, inclusive."
In both of these, the last line is indented, making it a code block. The fix is to unindent the lines.